pkgs/applications/terminal-emulators/rio/default.nix

```diff
@@ -0,0 +1,81 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+
+, autoPatchelfHook
+, cmake
+, pkg-config
+, python3
+
+, fontconfig
+, libGL
+, vulkan-loader
+
+, withX11 ? true
+, libX11
+, libXcursor
+, libXi
+, libXrandr
+, libxcb
+
+, withWayland ? false
+, wayland
+}:
+let
+  rlinkLibs = [
+    fontconfig
+    libGL
+    vulkan-loader
+  ] ++ lib.optional withX11 [
+    libX11
+    libXcursor
+    libXi
+    libXrandr
+    libxcb
+  ] ++ lib.optional withWayland [
+    wayland
+  ];
+in
+rustPlatform.buildRustPackage rec {
+  pname = "rio";
+  version = "0.0.6.1-unstable-2023-06-13";
+
+  src = fetchFromGitHub {
+    owner = "raphamorim";
+    repo = "rio";
+    rev = "fe44ac3a06cffa56c220688821c8e8eb1cad69b7";
```

Comment:
Before doing the next push please bump the commit to the latest

Reply:
Fixed.